---
layout: page
title: CV
permalink: /CV/
---

**Ted Laderas, PhD**

Postdoctoral Researcher 

Knight Cancer Institute

Instructor

Division of Bioinformatics and Computational Biology

Department of Medical Informatics and Clinical Epidemiology

Oregon Health & Science University

3181 SW Sam Jackson Park Road BICC

Portland, OR 97239

[laderast@ohsu.edu](mailto:laderast@ohsu.edu)


[Download a PDF version of my CV]({{site.url}}/assets/CV-laderas.pdf)

**EDUCATION**

B.A., Chemistry, Reed College, Portland, OR 1998.
M.S. Biomedical Informatics, Oregon Health & Science University (OHSU), 2004.
PhD, Biomedical Informatics, Oregon Health & Science University, 2014.

_Relevant Coursework_:

System Dynamics, Agent Based Modeling, Discrete Multivariate Modeling, Systems Biology, Biological Simulation, Systems Theory, Discrete Systems Simulation, Genetic Mechanisms, Development, Differentiation, and Cancer. Biochemical Structure and Function, Enzymology and Metabolism, Research Methods in Bioinformatics, Statistical Methods in Bioinformatics, Nucleic Acids and Information Flow, Advanced R/Bioconductor Programming, Introduction to Microarray Analysis, Intro to Computational Biology, Computational Biology, Bioinformatics I: Intro to Bioinformatics, Bioinformatics II: Topics in Bioinformatics, Bioinformatics III: Software Development in Bioinformatics, Machine Learning, Advanced Java Programming, Databases, Research Design

**PUBLICATIONS**

1. A Network-Based Model of Oncogenic Collaboration to Predict Drug Sensitivity. **Ted Laderas**, Laura Heiser and Kemal Sonmez. In press, submitted to Molecular Systems Biology.

2. The Consensus Molecular Subtypes of Colorectal Cancer. Justin Guinney, Rodrigo Dienstmann, Xin Wang, Aurélien de Reyniès, Andreas Schlicker, Charlotte Soneson, Laetitia Marisa, Paul Roepman, Gift Nyamundanda, Paolo Angelino, Brian M. Bot, Jeffrey S. Morris, Iris Simon, Sarah Gerster, Evelyn Fessler, Felipe de Sousa e Melo, Edoardo Missiaglia, Hena Ramay, David Barras, Krisztian Homicsko, Dipen Maru, Ganiraju C. Manyam, Bradley Broom, Valerie Boige, **Ted Laderas** , Ramon Salazar, Joe W. Gray, Douglas Hanahan, Josep Tabernero, Rene Bernards, Stephen H. Friend, Pierre Laurent-Puig, Jan P. Medema, Anguraj Sadanandam, Lodewyk Wessels, Mauro Delorenzi, Scott Kopetz, Louis Vermeulen, and Sabine Tejpar. Manuscript submitted to Nature Medicine.

3. Between Pathways and Networks lies Context. **Ted Laderas** and Shannon McWeeney. Review paper submitted to Science Progress. In press.

4. Computational detection of alternative exon usage. **Ted Laderas** , Nikki Walter, Michael Mooney, Kristina Vartanian, Priscila Darakjian, Kari Buck, Chris Harrington, John Belknap, Robert Hitzemann, and Shannon McWeeney.  Frontiers in Neurogenomics. 2011. Article 69. PMID 21625610

5. High throughput sequencing in mice: a platform comparison identifies a preponderance of cryptic SNPs. Nicole A Walter, Daniel Bottomly, **Ted Laderas** , Michael Mooney, Priscila Darakjian, Robert P Searles, Christina Harrington, Shannon K McWeeney, Robert Hitzemann, Kari J Buck. BMC Genomics. 2009 Aug 17;10:379.

6. Cross-platform Gene Expression Concordance Evaluation: A Tale of Mappings, Masks and Mice. Michael Mooney, **Ted Laderas** , Nikki Walter, Kristina Vartanian, Priscila Darakjian, Kari Buck, Chris Harrington, John Belknap, Robert Hitzemann, and Shannon McWeeney.  In press.

7. TandTRAQ: An open-source tool for integrated protein identification and quantitation. **Ted Laderas** , Cory Bystrom, Debra McMillen, Guang Fan and Shannon McWeeney.  Bioinformatics. 2007.

8. A consensus framework for clustering microarray data.   **Ted Ladera** s and Shannon McWeeney. OMICS: A Journal of Integrative Biology. OMICS: A Journal of Integrative Biology.  2007. 116-128.

9. Developing and Validating a Tool to Compare Microarray Clusterings. Masters Thesis, Biomedical Informatics, OHSU. 2004.

10. The Melting of Pulmonary Surfactant Monolayers. Wen-fei Yan, Samir C Biswas, **Ted Laderas** , Stephen B. Hall.  Journal of Applied Physiology. 2006. 1739-45.

11. Metastability of a supercompressed fluid monolayer. Ethan C. Smith, Jonathan M. Crane, **Ted G. Laderas** , and Stephen B. Hall. Biophysical Journal. 2003. 853048-57.

12. Persistence of Metastability after Expansion of a Supercompressed Fluid Monolayer. Ethan C. Smith, **Ted G. Laderas** , Jonathan M. Crane, and Stephen B. Hall. Langmuir. 2004. 204945 - 4953.

13. Resonance-Raman Spectroscopy, Chromium Hexacarbonyl, and Me: A Tale of Intrigue. Bachelor Thesis. Reed College. 1998.

**POSTERS/ABSTRACTS**

1. A Network-Based Model of Oncogenic Collaboration for Prediction of Drug Sensitivity. **Ted Laderas**. RECOMB/ISCB Conference on Regulatory and Systems Genomics, November 2014.

2. Integrating proteomics and genomic data to illuminate the effects of Lapatinib in HER2+ Cells. **Ted Laderas**. NLM Training Conference, Utah, 2013.

3. Integrating to Interact: Using Reconstructability Analysis to Integrate Genomic and Proteomic Data to Predict Drug Sensitivity. **Ted Laderas**. Intelligent Systems in Molecular Biology conference, 2012.

4. Tales from the Cryptic Snp: High Throughput Sequencing of DBA/2J and C57BL/6J mouse chromosome 1 reveals an abundance of polymorphisms. Kari J. Buck, Nicole A.R. Walter, Daniel Bottomly, **Ted Laderas** , Michale Mooney, Priscila Darakjian, Shannon McWeeney, Robert Hitzemann. Research Society on Alcoholism Annual Meeting. 2009.

5. A computational framework for the detection of alternative exon usage using Affymetrix Exon arrays. **Ted Laderas** , Michael Mooney, Nikki Walter, Kristina Vartanian, Christina Harrington, Priscila Darakijan, John Belknap, Robert Hitzemann, and Shannon McWeeney.  Intelligent Systems in Molecular Biology conference, Toronto, July 2008.

6. Cerebrospinal fluid (CSF) proteomics in children with acute lymphoblastic leukemia (ALL).  Robert C Trueworthy, Linda C Stork, Yanping Zhong, Sharon Pine, Yousif Matloub, **Ted Laderas** , Shannon McWeeney, and Guang Fan.  Poster 2006.

7. A metrics-based low-level preprocessing framework for proteomics data. **Ted Laderas** , Solange Mongoue-Tchokote, Veena Rajaraman, Kimberly Sellers, Cory Bystrom and Shannon McWeeney.  Bioconductor Developer's Meeting, Seattle, Washington, August 2006.

8. A diagnostic test suite for the RProteomics statistical routines.   **Ted Laderas** , Solange Mongoue-Tchokote, Veena Rajaraman, Kimberly Sellers and Shannon McWeeney.  caBIG annual meeting, Washington DC, April 2006.

9. Consense-Cluster: a framework for comparing and benchmarking clustering algorithms. **Ted Laderas** and Shannon McWeeney. Bioconductor Developer's Meeting, Seattle, Washington, August 2005.

**SOFTWARE (Sole Developer unless specified)**

1. R-Bootcamp: Introduction to R. Script-based introductory MOOC to R. [http://dx.doi.org/10.5281/zenodo.13756](http://dx.doi.org/10.5281/zenodo.13756)

2. Surrogate Mutation Explorer. R/Shiny Interactive Application for exploring surrogate mutations. [http://dx.doi.org/10.5281/zenodo.13757](http://dx.doi.org/10.5281/zenodo.13757)

3. SurrogateMutation. R Package for mapping mutations and copy number alterations to networks and associated statistics. [http://dx.doi.org/10.5281/zenodo.17303](http://dx.doi.org/10.5281/zenodo.17303)

4. Consense-Cluster. R Package for comparing clustering output across algorithms. [http://dx.doi.org/10.5281/zenodo.17304](http://dx.doi.org/10.5281/zenodo.17304)

5. ExonModelStrain. RPackage for detecting alternative exon usage between two strains of mice. [http://dx.doi.org/10.5281/zenodo.17305](http://dx.doi.org/10.5281/zenodo.17305)

6. TandTRAQ. Perl Script for mapping iTRAQ protein quantitation to X!Tandem peptides.

**PRESENTATIONS/LECTURES/WORKSHOPS**

1. Introduction to ggvis. Lecture/Workshop given for OHSU Bioinformatics Discussion for Students and Postdocs, April 2015.

2. Introduction to ggplot2. Lecture/Workshop given for OHSU Bioinformatics Discussion for Students and Postdocs, March 2015.

3. Everything you wanted to know about bioinformatics but were afraid to ask. Lecture/Workshop given for OHSU PhD/Postdoc Fellows meeting, October 2014.

4. Surrogate Mutations and Drug Sensitivity in Breast Cancer Cell Lines. Plenary talk at NLM Training Conference, Pittburgh June 2014.

5. Connecting Genotypes to Drug Sensitivities in HER2 Positive Cancer Cell Lines. Doctoral Defense, OHSU, March 2014.

6. Understanding Expression Differences in RPPA Data: Implications for ODE Models. Presentation at Integrative Cancer Biology Program retreat 2012.

7. Connecting Genotypes to Drug Sensitivity in HER2+ Breast Cancer Cell Lines. Lecture/Dissertation proposal given for PhD/Postdoc Meeting January 2012 and March 2012.

8. Connecting Genotypes to Drug Sensitivity in Breast Cancer Cell Lines. Presentation at Integrative Cancer Biology Program retreat 2011.

9. Work Smarter, Not Harder: Productivity Tools and You. Lecture given for PhD/Postdoctoral meeting, 2011.

10. What are Models For? Making Sense of Systems Biology. Predoctoral Symposium. 2010.

11. Extended Dependency Analysis. Presentation given for Information Theory Independent Study 2010.

12. Conant's _Laws of Information that Govern Systems_. Presentation given for Information Theory Independent Study, 2010.

13. Ashby's _Law of Requisite Variety_ and Conant's _Information Transfer in Regulation_. Presentation given for Information Theory Independent Study, 2010.

14. Two kinds of Robustness.  Presentation given to Bioinformatics Development Group, 2010.

15. Portland Alcohol Research Center Scientific Advisory Board Review: Bioinformatics & Statistics.  Presentation given to PARC SAB, OHSU, 2009.

16. An introduction to trend analysis and consensus modules for systems biology.  Presentation given with Sophia Jeng to Mathematical Modeling Core for Systems Virology, OHSU 2009.

17. An introduction to Consensus Modules for Systems Biology.  Lecture given with Sophia Jeng to Bioinformatics Development Group, OHSU, 2008.

18. An Introduction to Sloppy Systems Biology. Lecture given to Bioinformatics Development Group, OHSU, 2008.

19. Your in-silico lab notebook. Lecture given as part of Bioinformatics Software Development class, OHSU, 2008, 2009.

20. Simulated and Synthetic Data for Benchmarking Algorithms. Lecture given as part of Bioinformatics Methods class, OHSU, 2007.

21. Introduction to QPACA Pathway Analysis Software and BioPAX. Talk given at 2006 caBIG Annual Meeting, Washington DC.

22. Consense: A software package for utilizing multiple clustering methods on Microarray Data.  Talk given at OHSU Student Research Forum, 2005.

**TEACHING EXPERIENCE**

1. R-Bootcamp. Massive Open Online Course available at: [http://dx.doi.org/10.5281/zenodo.13756](http://dx.doi.org/10.5281/zenodo.13756)

2. List Comprehensions in Python. Lecture given for Bioinformatics Programming and Scripting Course, Fall 2012.

3. Introduction to Unit Testing. Lecture given for Bioinformatics Programming and Scripting Course, Fall 2012.

4. Introduction to Numerical Python (NumPy). Lecture given for Bioinformatics Programming and Scripting Course, Fall 2012.

5. Introduction to SciPy.  Lectures given for Bioinformatics Programming and Scripting Course, Fall 2012.

6. An Introduction to ODE Models. Lecture given for Systems Biology Class, 2012.

7. Integrating Data for Systems Biology.  Lecture given for Systems Biology Class, 2012.

8. Bayesian Networks.  Lecture given for Statistical Methods in Bioinformatics Class, 2011.

9. Gibbs Samplers. Lecture given for Statistical Methods in Bioinformatics Class, 2011.

10. Workshop on Strings and Matrices in R. Workshop given for Statistical Methods in Bioinformatics class. 2011.

11. Introduction to R Workshop.  Workshop given for Statistical Methods in Bioinformatics class. 2011.

12. Lab: Using Consense-Cluster to explore the Bittner dataset.  Laboratory given as part of Microarray Analysis Course, OHSU, 2006.

**EXPERIENCE**

_Research Assistant/Teaching Assistant_, Department of Chemistry, Reed College. 1998

 - Conducted research into multiphoton dissociation of Chromium Hexacarbonyl
 - Graded problem sets for professor
 - Monitored and tutored students in electronics laboratory

_Research Assistant/Computer Programmer_, Department of Molecular Medicine, Oregon Health & Science University. 1999-2001.

 - Conducted and collaborated on research about supercompressed films of lung surfactant
 - Maintained, documented, and redesigned instrument control programs
 - Responsible for supporting instrument control software in laboratory setting

_Teaching Assistant/Computer Programmer/Server Admin_, Department of Medical Informatics & Clinical Epidemiology, Oregon Health & Science University.  2001-2002.

 - Developed and Maintained Student Projects Database
 - Administered Apple XServe, installed and maintained Unix-Based applications

_Bioinformatics Developer/Project Manager_, OHSU Knight Cancer Institute, Oregon Health & Science University.  2003-2009.

- Developer/Documenter of Consense-Cluster (see achievements below)
- Project Manager for Cancer Biomedical Informatics Grid projects (caBIG)
- oversaw deliverables/managed three workers
- Software Tester/Code Reviewer for caBIG projects
- Documentation Developer/Editor for caBIG projects
- Liaison between Developer Institutions and National Cancer Institute (NCI)
- Developer/Interface Designer/Code Maintainer of Affymetrix Microarray Core Database (AMCDB)
- Developer/Code Reviewer for HAPPY eQTL project
- Developer for TandTRAQ project
- Primary Author on journal articles
- Planner of class curriculum for Systems Biology class
- Reviewer of journal articles
- Analyst/Bioinformatician/Developer on Exon Array project
- Analyst/Developer on Systems Biology project investigating Influenza and SARS

_NLM Predoctoral Fellow_, Medical Informatics and Clinical Epidemiology, Oregon Health & Science University 2009 – 2014

- Collaborated with Joe Gray's group for modeling project in Integrative Cancer Biology Program
- Worked within Paul Spellman's group to analyze breast cancer genomic data from The Cancer Genome Atlas
- Gained certification in modeling from PSU Systems Science Program
- Developed workflows to analyse both dynamic time course data and integrative genetic analysis
- Audited classes in Cancer Biology and Development and Signaling

_NLM Postdoctoral Fellow_, Oregon Health & Science University 2014-2015

- Developed Introductory Class in R programming as massive open online course
- Founder of Bioinformatics Discussion for Students and Postdocs (BioDSP)
- Developed visualization framework for Surrogate Mutations
- Analyzed surrogate mutations in 6 different cancer types

_Visiting Scientist_, Sage Bionetworks, Seattle Washington, 2014-2015

- Collaborated with Colorectal Cancer Consortium on biological interrogation of consensus subtypes
- Developed annotation pipeline to map proteomic data to genomic coordinates to enable comparison of cross

_Postdoctoral Researcher_, Knight Cancer Institute, 2015-Present

- Developing automated workflows for high-dimensional flow cytometry project

**RELATED ACHIEVEMENTS**

_Software/Algorithm Development/Data Analysis_

**Developed**** annotation pipeline** to map peptide data to genomic coordinates to enable comparison of alternative splicing in peptides versus transcriptional data. Expected result: investigational study comparing alternative splicing and isoform usage in colorectal cancer.

**Developed visualization framework** for evaluating oncogene collaboration using protein-protein interaction networks. Result: online application with interactive filtering and visualization tools available at

**Developed**  and **validating** analysis workflow for cell-line based mutation and copy number data to understand oncogene collaboration in a systems context.  _Expected Results_: workflow has potential to integrate personal mutation information into understanding of drug sensitivity.

**Developed** analysis workflow for Reverse Phase Protein Array (RPPA) timecourse data.  Developing and prototyping methods that allow for comparison of timecourse data across cell lines and visualize the data in a pathway-specific context.  _Expected Results_: workflow will enable researchers in Integrative Cancer Biology Program to understand gaps in current knowledge of pathways.

**Developed** analysis workflow for Affymetrix Mouse Exon arrays to detect alternative exon usage between two strains of mice.  Prototyped SNP masking workflow to allow for masking of strain/strain differences, remapped Affymetrix Probe sets to Ensembl Transcript structure, developed software package for analysis, wrote and published paper.  _Results:_ Software is freely available for everyone to analyse exon array data.  Paper is one of the most highly accessed articles for _Frontiers in Neurogenomics_.

**Developed** tool (TandTRAQ) for integrating protein database searching results with quantitative peptide analysis.  Reviewed current code and data formats, adapted code.  _Results_: tool produced list of peptides of interest for follow up in cerebrospinal fluid (CSF).  Paper on TandTRAQ has been published.

**Developed** and **redesigned** webpage interface of Affymetrix Microarray Core (AMC) database.  Explored current applications available for annotating data, designed improved interface, presented improved interface to AMC database to users.  Results: positive feedback from users with respect to new GUI.

**Designed** , **developed** , and **validated** tool (Consense-Cluster) to compare results for microarray cluster analysis using R/Bioconductor for Master's thesis.  Conducted literature review, designed program structure, coded program in R, ran simulated and real microarray data in program, validated results using annotations. _Results_: tool will be released as an open-source Bioconductor project, providing biologists with the ability to compare clustering results.

**Utilized** Consense-Cluster to explore leukemia phosphopeptide dataset.  Explored dataset using multiple clustering methods, grouped samples into two groupings.  _Results_: clustering reproduced patient/control groupings blinded, produced peptide list for possible follow up.

**Reviewed** , **annotated** and **modified** Happy eQTL code to run permutation analysis.  Reviewed and commented entire codebase, designed permutation analysis component, integrated code modifications into analysis.  _Result_: eQTL permutation analysis allowed researchers to assign p-value to QTL analysis.

_Software Evaluation/Testing_

**Designed** and **utilized** synthetic proteomics dataset for benchmarking classification algorithms.  Result: validation of caBIG related proteomics application.

**Evaluated** , **reviewed** and **debugged** caBIG application for classifying proteomics data.  Reviewed code, compared approach to other classification algorithms, developed a testing framework for benchmarking classification algorithms.  _Result_: current version of caBIG application has improved classification ability.

**Reviewed** and **tested** code for caBIG RProteomics statistical framework.  Extensively reviewed entire code framework and evaluated effectiveness of statistical routines.  _Results_: Several bugs in implementation were found; testing results were submitted for future improvements to the RProteomics project.

_Project Management_

**Managed** and **produced** project deliverables for 3 caBIG applications.  Developed documentation, test plans, project time, assigned tasks for 3 different people, managed deliverable timeline.  _Result_: all deliverables were approved and delivered within assigned timeframe.

_Academic_

**Founded student discussion group for bioinformatics students and postdocs**. Discussion group is now active and students are more actively enaged.

**Tutored and mentored students in R programming.** Actively taught students in one on one sessions, helping them debug code.

**Developed workshop/massively open online course introducing R programming**.  Researched topics, put together self-teaching workshop materials, oversaw workshop.

**Delivered two lectures** for Systems Biology class. Researched topics (multi-omic integration, ODE modeling), developed lectures, and moderated student discussions.

**Delivered four lectures** for Bioinformatics Programming and Scripting class.  Researched topics (List comprehensions, Unit Testing, Numerical Python, and Scientific Python), developed lectures, oversaw in class exercises, and tutored students one on one.

**Delivered two lectures** in Statistical Methods in Bioinformatics Class.  Researched topics (Bayesian Networks and Gibbs Samplers for Motif Finding), developed lectures, and delivered in class.

**Helped write** and **plan** class curriculum for Systems Biology/Network Reconstruction class.  Read primary literature, summarized textbook for class notes, developed examples.  _Result_: class curriculum was used for course in Spring 2007.

**Documented** and **wrote** paper on utility of Consense-Cluster.  Explored primary literature, wrote paper, developed supplementary website.  Result: Paper is currently published.

**Reviewed** journal articles for _Neurocomputing _and _BMC Bioinformatics_.

**AWARDS**

NLM Predoctoral Fellowship. 2009-2013.

NLM Postdoctoral Fellowship. 2014-2015.

**REFERENCES**

Dr. Shannon McWeeney, Oregon Health & Science University, [mcweeney@ohsu.edu](mailto:mcweeney@ohsu.edu)

Dr. Laura Heiser, Oregon Health & Science University, [heiserl@ohsu.edu](mailto:heiserl@ohsu.edu)

Dr. Wayne Wakeland, Portland State University, [wakeland@pdx.edu](mailto:wakeland@pdx.edu)

Dr. Robert Hitzemann, Oregon Health & Science University, [hitzeman@ohsu.edu](mailto:hitzeman@ohsu.edu)

Dr. Stephen Hall, Oregon Health & Science University, [sbh@ohsu.edu](mailto:sbh@ohsu.edu)

Dr. Daniel Gerrity, Reed College, [gerrity@reed.edu](mailto:gerrity@reed.edu)
