---
layout: default
title: Homepage of Ted Laderas, PhD
---

![Ted Image]({{site.url}}/images/ted-headshot.jpg)

I am a Postdoctoral Researcher at the [OHSU Knight Cancer Institute](http://www.ohsu.edu/xd/health/services/cancer/) and an instructor
in the Bioinformatics and Computational Biology track in the OHSU Department of
Medical Informatics and Clinical Epidemiology ([BCB/DMICE](http://www.ohsu.edu/xd/education/schools/school-of-medicine/departments/clinical-departments/dmice/educational-programs/dmice-programs/computational-biology.cfm)). My research focus is on the Systems Biology of Complex Diseases. I use integrative modeling approaches across OMICs types to achieve this.

Over twelve years of bioinformatics experience, I have developed methods for low-level preprocessing and high-level analysis of many OMICs types, including expression (especially alternative splicing), phosphoproteomics, proteomics, and genomics. My experience in functional genomics has led me to use models such as Graphical Models and Networks to integrate these many OMICs types into building a unified picture of integrated systems-level dysregulation in complex diseases such as alcoholism and cancer.

## **Research Interests**

+  Phenotypic Heterogeneity of Cancer and Measurement using Flow Cytometry Methods
+  Systems Biology of Cancer, especially oncogenic collaboration
+  Prioritization of Combination Therapies in the treatment of cancer
+  Integrating multi-OMICs data to understand dysregulation in cancer
+  Modeling Disease using Graphical Models and Agent-Based Models
+  Interactive visualization of Data

## **Teaching and Mentoring Interests**

I have developed a number of self-directed training materials for learning R and specific packages that I've found useful. Most of these are available from my github page and may be used freely by students.

+ [R-Bootcamp](https://www.coursesites.com/s/_Rbootcamp) - an MOOC (Massively Open Online Course) for teaching the basics of data manipulation in R (Coursesites registration required).

### **Standalone Tutorials**

Many of these tutorials have been given through the student club at OHSU that I mentor, called[BioData Club](http://biodata-club.github.io). They are all freely available to be reused by other instructors.

+ [The Magic of Markdown](https://github.com/laderast/magic-of-markdown) - An introduction to the many uses of Markdown, including RMarkdown for reproducible scripting.
+ [GitHub Pages](https://github.com/BioData-Club/githubPagesTutorial) - A pain free introduction to getting a personal website started using GitHub pages. With Robin Champieux and Eric Leung

#### **R Tutorials**

As a developer and programmer of R, I enjoy teaching others about the basics of data wrangling and analysis. Here are a few of the tutorials I've put together.

+ [Introduction to the Tidyverse](https://github.com/erictleung/tutorial-tidyverse) - An introduction to basic data wrangling using the Tidyverse in R. With Eric Leung.
+ [ggplot2 Tutorial](https://github.com/laderast/ggplot2-intro) - A gentle introduction to ggplot2 and the grammar of graphics.
+ [ggvis Tutorial](https://github.com/laderast/ggvis-intro) - A gentle introduction to ggvis, which adds extra interactivity to Shiny plots using the magic of vega.
+ [Shiny Tutorial](https://github.com/laderast/shinyTutorial) - A do it yourself tutorial to try out Shiny, ggplot2, and dplyr for interactive graphics.
+ [igraph Tutorial](https://github.com/laderast/igraphTutorial) - Another tutorial, this time on network analysis using the igraph package.
+ [Exacloud Tutorial](https://www.dropbox.com/s/chg6ciknxonp5el/exacloud-tutorial.pdf) - A DIY tutorial to running jobs on Exacloud, OHSU's cluster computing environment. With Ryan Swan.

#### **Big Data to Knowledge**

As an instructor under OHSU's [Big Data To Knowledge (BD2K)](http://www.ohsu.edu/xd/education/schools/school-of-medicine/departments/clinical-departments/dmice/research/bd2k.cfm) Training Grant, I have also developed the following short workshops to encourage students to explore data and learn the basics of data-wrangling.

+ [Exploratory Data Analysis Using Shiny](https://github.com/laderast/shinyEDA) - A Tutorial and slides about doing exploratory data analysis with a Shiny Dashboard
+ [Machine Learning Using RMarkdown](https://github.com/laderast/MLtutorial) - A tutorial about Machine Learning and Reproducible Scripting and Workflows.

#### **Other Teaching Interests**

+ [BioData Club](http://biodata-club.github.io) - A student and postdoc driven discussion group at OHSU that focuses on practical skills (documentation, software engineering and visualization) necessary for success in data science and bioinformatics.
+ I'm also an Instructor for the summer hybrid Analytics Course taught at OHSU (Hybrid course co-taught with Kaiser Permanente Data & Information Management Enhancement (DIME) group).
+ I also participate in the [PDX R User Group](https://www.meetup.com/portland-r-user-group/) and I'm helping organize the first [Cascadia R conference](https://cascadiarconf.com)

### **Software Development:**

+ Software Reporting for automating and visualizing CyTOF gating results (to be published soon).
+ [surrogateMutation](https://github.com/laderast/surrogateMutation) - a package for detecting oncogenic collaboration in somatic mutation and copy nubmer data.
+ [surrogateShiny](https://github.com/laderast/surrogateShiny) - an R/Shiny Framework for exploring oncogenic collaboration in breast cancer cell lines. [See it in action.](https://tladeras.shinyapps.io/surrogateShiny/)
+ [Consense](https://github.com/laderast/surrogateMutation) - a package for comparing clustering methods.
+ [ExonModelStrain](https://github.com/laderast/ExonModelStrain) - a package for detecting alternative exon usage in the Affymetrix Exon Array
+ TandTRAQ - Perl Script for merging iTRAQ and XTandem Results.

### **Selected Publications:**

+ [A Network Based Model of Oncogenic Collaboration to Predict Drug Sensitivity.](http://journal.frontiersin.org/article/10.3389/fgene.2015.00341/abstract) **Ted Laderas**, Kemal Sonmez, and Laura Heiser. Frontiers in Genetics. 2015.
+ [Between Networks and Pathways Lies Context.](http://www.researchgate.net/publication/281769232_Between_pathways_and_networks_lies_context_Implications_for_precision_medicine) **Ted Laderas**, Guanming Wu, and Shannon McWeeney. Science Progress. 2015.
+ [The Consensus Molecular Subtypes of Colorectal Cancer](http://www.nature.com/nm/journal/vaop/ncurrent/full/nm.3967.html). Justin Guinney, Rodrigo Dienstmann, Xin Wang, Aurélien de Reyniès, Andreas Schlicker, Charlotte Soneson, Laetitia Marisa, Paul Roepman, Gift Nyamundanda, Paolo Angelino, Brian M. Bot, Jeffrey S. Morris, Iris Simon, Sarah Gerster, Evelyn Fessler, Felipe de Sousa e Melo, Edoardo Missiaglia, Hena Ramay, David Barras, Krisztian Homicsko, Dipen Maru, Ganiraju C. Manyam, Bradley Broom, Valerie Boige, **Ted Laderas**, Ramon Salazar, Joe W. Gray, Douglas Hanahan, Josep Tabernero, Rene Bernards, Stephen H. Friend, Pierre Laurent-Puig, Jan P. Medema, Anguraj Sadanandam, Lodewyk Wessels, Mauro Delorenzi, Scott Kopetz, Louis Vermeulen, and Sabine Tejpar. Nature Medicine. 2015.
+ [Computational detection of alternative exon usage.](http://journal.frontiersin.org/article/10.3389/fnins.2011.00069/abstract) **Ted Laderas**, Nicole Walter, Michael Mooney, Kristina Vartanian, Priscila Darakjian, Kari Buck, Chris Harrington, John Belknap, Robert Hitzemann, and Shannon McWeeney. Frontiers in Neurogenomics. 2011. Article 69. PMID 21625610. In this paper, Nicole Walter and I developed a computational framework for detecting alternative exon usage between strains of mice (B6/alcohol preferring and D2/wild type).
+ [TandTRAQ: An open-source tool for integrated protein identification and quantitation.](http://bioinformatics.oxfordjournals.org/content/23/24/3394.full) **Ted Laderas**, Cory Bystrom, Debra McMillen, Guang Fan and Shannon McWeeney.  Bioinformatics. 2007.
+ [A consensus framework for clustering microarray data.](http://www.ncbi.nlm.nih.gov/pubmed/17411399) **Ted Laderas** and Shannon McWeeney. OMICS: A Journal of Integrative Biology. 2007. 116-128.
