\documentclass[11pt,]{article}
\usepackage[sc, osf]{mathpazo}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1in]{geometry}
\usepackage{hyperref}
\PassOptionsToPackage{usenames,dvipsnames}{color} % color is loaded by hyperref
\hypersetup{unicode=true,
            pdftitle={Ted Laderas, PhD},
            pdfkeywords={R Markdown, academic CV, template},
            colorlinks=true,
            linkcolor=blue,
            citecolor=Blue,
            urlcolor=blue,
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}

  \title{Ted Laderas, PhD}
    \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
    \author{}
    \preauthor{}\postauthor{}
    \date{}
    \predate{}\postdate{}
  

\begin{document}
\maketitle

\emph{Assistant Professor}\\
Division of Bioinformatics and Computational Biology\\
Department of Medical Informatics and Clinical Epidemiology

\emph{Researcher}\\
OHSU Knight Cancer Institute

Oregon Health \& Science University\\
3181 SW Sam Jackson Park Road CR145\\
Portland, OR 97239\\
(503) 481-8470\\
\href{mailto:laderast@ohsu.edu}{\nolinkurl{laderast@ohsu.edu}}\\
\href{mailto:tedladeras@gmail.com}{\nolinkurl{tedladeras@gmail.com}}\\
\url{http://laderast.github.io}

\section{EDUCATION}\label{education}

B.A., Chemistry, Reed College, Portland, OR \hfill 1998.\\
M.S. Biomedical Informatics, Oregon Health \& Science University,
Portland, OR \hfill 2004.\\
PhD, Biomedical Informatics, Oregon Health \& Science University,
Portland, OR \hfill 2014.

\section{TEACHING}\label{teaching}

\subsection{Teaching Statement}\label{teaching-statement}

Effective communication, hands-on learning, and mentoring are core to my
teaching philosophy. I am a strong believer in enabling students in
self-directed learning through hands-on workshops including both
practical skills in bioinformatics (software development, design, and
data analysis and visualization) and social skills essential to
collaboration (interpretation and communication). In terms of practical
skills, I have developed and contributed to a number of courses
(Analytics, RBootcamp, Network Analysis, and BD2K skills courses) and
workshops in a large variety of software development and communication
topics. These topics include software design, interactive visualization,
exploratory data analysis, and presentation of results. In terms of
fostering social and collaborative skills in students, I am a founder
and co-leader of \href{https://biodata-club.github.io}{BioData-Club}, a
collaborative forum for students and postdocs to share and teach each
other practical skills for success in a research environment. Through
BioData Club, I have informally mentored a number of students through
feedback and collaboration on workshops and presentations. I believe
that by ``training the trainers'', we can more effectively educate
students in these skills in Data Science and Analytics. In order to
accomplish this training of trainers, my course materials are openly
available for use and modification by other teachers. Because of my
dedication to teaching and incorporation of student feedback, I have
consistently high ratings from students in terms of teaching both
long-form courses and short-term workshops.

\subsection{Current Academic Courses}\label{current-academic-courses}

\begin{itemize}
\tightlist
\item
  \emph{\href{https://github.com/laderast/AnalyticsCourse}{BMI569/669
  Data Analytics}}, co-taught with Kaiser Permanente (Brian Sikora and
  Delilah Moore). Summer Quarter, 2015-Present.
\item
  \emph{\href{https://github.com/dasaderi/python_neurobootcamp}{NEUS640
  Python Bootcamp for Neuroscientists}.} Taught with Stephen David and
  Brad Buran. Winter Quarter, 2018.
\item
  \emph{BMI551/561 BCBII Statistical Methods in Computational Biology}.
  Co-instructor. Winter Quarter, 2014-Present.
\item
  \emph{\href{http://laderast.github.io/HSMP410/}{HSMP/PHE410 Health
  Informatics}.} Course Co-Director. Spring Quarter, 2018.
\item
  \emph{BMI 533/633 Data Harmonization and Standards for Translational
  Research}. Course to be taught Spring Quarter, 2019. Course is
  co-designed with Christina Zheng and Melissa Haendel.
\end{itemize}

\subsection{Courses in which I have guest
taught}\label{courses-in-which-i-have-guest-taught}

\begin{itemize}
\tightlist
\item
  BMI523 Clinical Research Informatics (2017)
\item
  BMI567 Network Science and Biology (2015, 2016, 2017)
\item
  BMI535 Management and Processing of Large Scale Data (2017)
\item
  CSE631 Data Visualization (2017, 2018)
\item
  OHSU Data Science Institute (2017)
\item
  BioData Club (2015-Present)
\end{itemize}

\subsection{Teaching materials and
workshops}\label{teaching-materials-and-workshops}

Lectures and Interactive Workshops are linked where possible.

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \href{https://cascadiarconf-wrangle.netlify.com}{A gRadual
  intRoduction to Data wRangling}. Workshop for Cascadia R. June 2018.
  Ted Laderas and Chester Ismay.
\item
  \href{https://cascadiarconf-viz.netlify.com}{A gRadual intRoduction to
  Data Visualization}. Workshop for Cascadia R. June 2018. Chester Ismay
  and Ted Laderas.
\item
  \href{http://tladeras.shinyapps.io/dataLiteracy/}{Data Literacy}.
  Based on \href{http://github.com/laderast/DSIExplore}{DSIExplore} (by
  myself and Jessica Minnier) and
  \href{https://github.com/bearloga/wmf-allhands18}{wmf-allhands18} by
  Mikhail Popov.
  \href{https://github.com/laderast/dataLiteracyTutorial}{GitHub repo}
  (installable as a LearnR package).
\item
  \href{http://laderast.github.io/HMSP410}{\emph{HMSP410: Health
  Informatics}}. Course material for HMSP410 Health Informatics course
  at OHSU/PSU School of Public Health. Spring 2018.
\item
  \href{https://laderast.github.io/exacloud_tutorial/}{\emph{Exacloud
  Training Workshop}}. Guest lecture for BMI535 Management and
  Processing of Large Scale Data. Winter 2018.
\item
  \href{https://laderast.github.io/gradual_shiny/}{\emph{A gRadual
  intRoduction to Shiny}}. Workshop for Portland R user group. Winter
  2018. Also given for CSE631, Data Visualization.
\item
  \href{https://laderast.github.io/sysc_data_sci/}{\emph{How are Systems
  Science and Data Science Connected?}}. Seminar for Portland State
  Systems Science Program. Winter 2018.
\item
  \href{https://github.com/dasaderi/python_neurobootcamp}{\emph{NEUS640
  Python Bootcamp For Neuroscientists}}. Ted Laderas, Brad Buran,
  Daniela Sadieri, Charles Heller, Michael Mooney, Lisa Karstens, and
  Stephen David. 5 day in-person workshop for introductory Python using
  Neuroscience data. Winter 2018. Role: course coordinator.
\item
  \href{https://laderast.github.io/BMI523slides/}{\emph{BMI 523:
  Actionable Gene Variants}}. Guest Lecture for BMI 523 Clinical
  Research Informatics. November 2017. Role: Guest Lecturer
\item
  \href{https://github.com/laderast/DSIExplore}{\emph{Exploratory Data
  Analysis and Statistics}}. Workshop with Jessica Minnier for OHSU Data
  Science Institute 2017. Role: instructor.
\item
  \href{https://www.datacamp.com/courses/rbootcamp}{\emph{R-Bootcamp
  (tidyverse version on DataCamp)}}. Active exercises for learning
  introductory R using the tidyverse. Written with Jessica Minnier and
  Chester Ismay. 2017. Code and teaching material available at
  \url{https://github.com/laderast/RBootcamp} Role: course developer and
  instructor.
\item
  \href{https://github.com/Cascadia-R/gRadual-intRoduction-tidyverse}{\emph{A
  gRadual intRoduction to the \texttt{tidyverse}}}. Workshop given with
  Chester Ismay for \href{https://cascadiarconf.com}{Cascadia-R 2017}
  introducing visualization and data cleaning using the
  \texttt{tidyverse}.
\item
  \href{https://github.com/laderast/cvdNight1}{\emph{Assessing
  Cardiovascular Risk}}. 2 night Workshop for Portland State University
  Students teaching
  \href{https://github.com/laderast/cvdNight1}{Exploratory Data
  Analysis} and \href{https://github.com/laderast/cvdNight2}{Machine
  Learning} on a synthetic patient cohort. With David Dorr. May 2017.
\item
  \href{https://github.com/biodata-club/githubPagesTutorial}{\emph{An
  Intro to GitHub Pages}}. Workshop given with Robin Champieux and Eric
  Leung on setting up a personal GitHub webpage. April 2017, and May
  2018.
\item
  \href{https://github.com/erictleung/tutorial-tidyverse}{\emph{An Intro
  To Data Carpentry}}. Lecture given with Eric Leung about the tidyverse
  suite of packages for data wrangling and visualization. March 2017.
\item
  \href{https://github.com/laderast/magic-of-markdown}{\emph{The Magic
  of Markdown}}. Updated for 2017, including examples of using Zotero as
  citation manager. March 2017.
  \href{https://doi.org/10.5281/zenodo.495614}{doi:10.5281/zenodo.495614}
\item
  \href{https://github.com/laderast/CSE631Shiny}{\emph{Shiny Tutorial
  for CSE631}}. Workshop on developing interactive visualizations using
  Shiny for CSE631 Data Visualization course. November 2016.
  \href{https://doi.org/10.5281/zenodo.495621}{DOI:
  10.5281/zenodo.495621}
\item
  \href{https://github.com/laderast/AnalyticsCourse}{\emph{BMI569/669
  Analytics}}. New Course material in SQLite, Logistic Regression and
  Analysis. August 2016.
  \href{https://doi.org/10.5281/zenodo.495623}{doi:10.5281/zenodo.495623}
\item
  \href{https://github.com/laderast/DreamEDAShiny}{\emph{Exploring the
  DREAM Viral Respiratory Dataset using Shiny}}. Tutorial in Exploratory
  Data Analysis (EDA) using \emph{data.table} and \emph{Shiny}. July
  2016.
\item
  \href{https://github.com/laderast/shinyEDA}{\emph{Introduction to
  Exploratory Data Analysis using Shiny}} - Interactive workshop in
  using a Shiny Dashboard to conduct EDA on a dataset for BD2K Advanced
  Skills Course. May 2016.
  \href{https://doi.org/10.5281/zenodo.495618}{doi:10.5281/zenodo.495618}
\item
  \href{https://github.com/laderast/MLtutorial}{\emph{Introduction to
  Machine Learning using Markdown}} - Interactive workshop using
  Markdown to explore machine learning algorithms for BD2K Advanced
  Skills Course. May 2016.
\item
  \emph{Clustering Algorithms}. Lecture given for Statistical Methods
  class. February 2016.
\item
  \href{https://github.com/laderast/igraphTutorial}{\emph{iGraph
  Tutorial}}. Introductory lecture for the igraph package in R for
  network analysis course. November 2015.
  \href{https://doi.org/10.5281/zenodo.495616}{doi:10.5281/zenodo.495616}
\item
  \href{https://github.com/laderast/magic-of-markdown}{\emph{The Magic
  of Markdown}}. Introduction to Markdown in both R, GitHub pages, and
  Pandoc. BioDSP October 2015.
  \href{https://doi.org/10.5281/zenodo.495614}{doi:10.5281/zenodo.495614}
\item
  \emph{Pharmacogenomics} Lecture for Analytics Course, August 2015.
\item
  \href{http://church.ohsu.edu:3838/laderast/clusteringLecture/}{\emph{Introduction
  to Clustering}}\emph{.} Interactive slides for understanding
  clustering.
  \href{https://doi.org/10.5281/zenodo.495624}{doi:10.5281/zenodo.495624}
\item
  \href{https://www.dropbox.com/s/n0unpdkd5r2tdu7/ggvis-Tutorial.html?dl=0}{\emph{Introduction
  to ggvis}}. Lecture/Workshop given for OHSU Bioinformatics Discussion
  for Students and Postdocs, April 2015.
\item
  \href{https://www.dropbox.com/s/efbwx3jjx89folh/ggplot2Intro.html?dl=0}{\emph{Introduction
  to ggplot2}}\emph{.} Lecture/Workshop given for OHSU Bioinformatics
  Discussion for Students and Postdocs, March 2015.
  \href{https://doi.org/10.5281/zenodo.495622}{doi:10.5281/zenodo.495622}
\item
  \href{https://www.dropbox.com/s/chg6ciknxonp5el/exacloud-tutorial.pdf}{\emph{Exacloud
  Tutorial}} - A DIY tutorial to running jobs on Exacloud, OHSU's
  cluster computing environment. November 2015.
\item
  \href{https://www.dropbox.com/s/5ceg6wdrustjoae/tutorialApp.zip}{\emph{Shiny
  Tutorial}} - A do it yourself tutorial to try out Shiny, ggplot2, and
  dplyr for interactive graphics. September 2015.
  \href{https://doi.org/10.5281/zenodo.495620}{doi:10.5281/zenodo.495620}
\item
  \emph{Analytics Course}. Instructor. Hybrid Online/On-campus joint
  course with DMICE and Kaiser Permanente Analytics group. August 2015.
\item
  \emph{R-Bootcamp}. Massively Open Online Course available at:
  \url{http://dx.doi.org/10.5281/zenodo.13756}. With contributions from
  Eric Leung, Dian Chase, Tracy Edinger, Clint Olson, and Gabrielle
  Chonoo. 2014-Present
\item
  \href{https://www.dropbox.com/s/ddyq88gqh8priv8/insilicotalk-2015.pptx?dl=0}{\emph{Your
  In-silico Lab Notebook: Best Practices 2015}}\emph{.} Lecture/Workshop
  given for OHSU BioDSP group, January 2015.
\item
  \href{https://www.dropbox.com/s/a4qgwap4jehzs8x/everything-you-wanted-to-know.pptx?dl=0}{\emph{Everything
  you wanted to know about bioinformatics but were afraid to
  ask}}\emph{.} Lecture/Workshop given for OHSU PhD/Postdoc Fellows
  meeting, October 2014.
\item
  \href{https://www.dropbox.com/s/1my6l8diprnzg4n/lecture_slides_day7_2012-laderas.pptx?dl=0}{\emph{List
  Comprehensions in Python}}. Lecture given for Bioinformatics
  Programming and Scripting Course, Fall 2012.
\item
  \href{https://www.dropbox.com/s/jdgbb07xydmxah5/lecture_slides_day10_1027-laderas.pdf?dl=0}{\emph{Introduction
  to Unit Testing}}. Lecture given for Bioinformatics Programming and
  Scripting Course, Fall 2012.
\item
  \emph{Introduction to Numerical Python (NumPy).} Lecture given for
  Bioinformatics Programming and Scripting Course, Fall 2012.
\item
  \emph{Introduction to SciPy.} Lectures given for Bioinformatics
  Programming and Scripting Course, Fall 2012.
\item
  \emph{An Introduction to ODE Models.} Lecture given for Systems
  Biology Class, 2012.
\item
  \href{https://www.dropbox.com/s/ksnogpxoh22eit8/IntegratingDataforSystemsBiology.pptx?dl=0}{\emph{Integrating
  Data for Systems Biology}}. Lecture given for Systems Biology Class,
  2012.
\item
  \emph{Work Smarter, Not Harder: Productivity Tools and You.} Lecture
  given for PhD/Postdoctoral meeting, 2011.
\item
  \href{https://www.dropbox.com/s/xab61u1jo9y9io9/bayesnets-new-laderas.pptx?dl=0}{\emph{Bayesian
  Networks}}. Lecture given for Statistical Methods in Bioinformatics
  Class, 2011.
\item
  \href{https://www.dropbox.com/s/r2j0oi1madn0t1s/GibbsSampler-laderas.pptx?dl=0}{\emph{Gibbs
  Samplers}}. Lecture given for Statistical Methods in Bioinformatics
  Class, 2011.
  \href{https://doi.org/10.6084/m9.figshare.4829530}{doi:10.6084/m9.figshare.4829530}
\item
  \emph{Workshop on Strings and Matrices in R}. Workshop given for
  Statistical Methods in Bioinformatics class. 2011.
\item
  \emph{Introduction to R Workshop.} Workshop given for Statistical
  Methods in Bioinformatics class. 2011.
\item
  \href{https://www.dropbox.com/s/yt8dybaap4sn8hy/03-eda-extended-dependency-analysis.pptx?dl=0}{\emph{Extended
  Dependency Analysis}}. Presentation given for Information Theory
  Independent Study 2010.
\item
  \href{https://www.dropbox.com/s/kkv4yv2vmwwdqfa/02-conant-laws-of-information.pptx?dl=0}{\emph{Conant's
  Laws of Information that Govern Systems}}. Presentation given for
  Information Theory Independent Study, 2010.
\item
  \href{https://www.dropbox.com/s/jz0gthjbd9cqbrt/01-ashby-conant-variety.pptx?dl=0}{\emph{Ashby's
  Law of Requisite Variety and Conant's Information Transfer in
  Regulation}}. Presentation given for Information Theory Independent
  Study, 2010.
\item
  \emph{Lab: Using Consense-Cluster to explore the Bittner dataset.}
  Laboratory given as part of Microarray Analysis Course, OHSU, 2006.
\end{enumerate}

\section{RESEARCH}\label{research}

\subsection{Research Statement}\label{research-statement}

My research interests are complex diseases, precision medicine,
applications of systems science (including network analysis and
modeling), and applying data integration to difficult and high-impact
translational research questions. These questions include immune system
profiling in both infectious disease (tuberculosis) and Acute Myeloid
Leukemia, understanding drug sensitivity in the context of multiple
cancer types (AML, Colorectal, Breast and Head and Neck Cancer), and
quantifying expression differences in alcoholic preference. I have
worked with a large number of datatypes (high-throughput
immunophenotyping, proteomics, expression, genomic, and functional drug
screen data) and have focused on methods and frameworks integrating
these datatypes within the biological and clinical context of these
translational research questions. My training in biomedical informatics
as a master's student in Biomedical Informatics, as an NLM Predoctoral
Fellow, and as a NLM Postdoctoral fellow has enabled me to communicate
with a wide variety of collaborators by giving me a strong background in
Cancer Biology, Software Development, and Clinical Systems.
Additionally, I am a strong advocate for Open Science initiatives, most
notably the effort for reproducibility in scientific analysis. To this
end, I have developed multiple novel software pipelines that
transparently process data from raw data to through the final stages of
analysis.

\subsection{Selected Research
Publications}\label{selected-research-publications}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \emph{Teaching data science fundamentals through realistic synthetic
  clinical cardiovascular data.} \textbf{Ted Laderas}, Nicole
  Vasilevsky, Bjorn Pederson, Shannon McWeeney, Melissa Haendel, and
  David Dorr. Submitted to JAMIA. BioRxiv Link:
  \url{https://www.biorxiv.org/content/early/2017/12/12/232611}. 2017.
  Contribution: First author: helped conceive study, designed bayesian
  network, developed course material based on dataset.
\item
  \emph{Integrated functional and mass spectrometry-based flow
  cytometric phenotyping to describe the immune microenvironment in
  acute myeloid leukemia}. Adam J Lamble, Matt Dietz, \textbf{Ted
  Laderas}, Shannon McWeeney, Evan F Lind. Journal of Immunological
  Methods. 2017. Contribution: developed informatics and visualization
  methodology.
\item
  \emph{Probabilistic Graphical Models for Systems Biology}. \textbf{Ted
  Laderas}, Aurora Blucher, Guanming Wu and Shannon McWeeney. Awaiting
  submission. Contribution: first author and conception of manuscript.
\item
  \href{http://journal.frontiersin.org/article/10.3389/fgene.2015.00341/abstract}{\emph{A
  Network-Based Model of Oncogenic Collaboration to Predict Drug
  Sensitivity}.} \textbf{Ted Laderas}, Laura Heiser and Kemal Sonmez.
  Featured article in Frontiers in Genetics. 2015. Contribution: First
  author, developer of methodology and visualization framework.
\item
  \href{http://www.bloodjournal.org/content/128/22/2829?sso-checked=true}{\emph{Mass
  Cytometry As a Modality to Identify Candidates for Immune Checkpoint
  Inhibitor Therapy within Acute Myeloid Leukemia}}. Adam Lamble, Yoko
  Kosaka, Fei Huang, A Kate Sasser, Homer Adams III, Cristina E. Tognon,
  \textbf{Ted Laderas}, Shannon K. McWeeney, Marc M. Loriaux, Jeffrey W.
  Tyner, Brian J. Druker and Evan F. Lind. Blood, 2016. Contribution:
  contributed to panel development of CyTOF.
\item
  \href{https://www.dropbox.com/s/p92dn5buctlyqpc/nm.3967.pdf?dl=0}{\emph{The
  Consensus Molecular Subtypes of Colorectal Cancer}.} 123 citations,
  Impact Factor 30.36. Justin Guinney, Rodrigo Dienstmann, Xin Wang,
  Aurélien de Reyniès, Andreas Schlicker, Charlotte Soneson, Laetitia
  Marisa, Paul Roepman, Gift Nyamundanda, Paolo Angelino, Brian M. Bot,
  Jeffrey S. Morris, Iris Simon, Sarah Gerster, Evelyn Fessler, Felipe
  de Sousa e Melo, Edoardo Missiaglia, Hena Ramay, David Barras,
  Krisztian Homicsko, Dipen Maru, Ganiraju C. Manyam, Bradley Broom,
  Valerie Boige, \textbf{Ted Laderas}, Ramon Salazar, Joe W. Gray,
  Douglas Hanahan, Josep Tabernero, Rene Bernards, Stephen H. Friend,
  Pierre Laurent-Puig, Jan P. Medema, Anguraj Sadanandam, Lodewyk
  Wessels, Mauro Delorenzi, Scott Kopetz, Louis Vermeulen, and Sabine
  Tejpar. Nature Medicine. 2015. Contribution: mapped and analyzed OMICs
  data to consensus cancer subtypes.
\item
  \href{https://www.dropbox.com/s/70z4yp3k36y9bmu/laderast_between_pathways.pdf?dl=0}{\emph{Between
  Pathways and Networks lies Context}}. Impact Factor: 1.76. \textbf{Ted
  Laderas}, Guanming Wu, and Shannon McWeeney. Science Progress. 2015.
  Contribution: first author, conceptualized and wrote manuscript.
  \href{http://dx.doi.org/10.3184/003685015X14368898634462}{doi:10.3184/003685015X14368898634462}
\item
  \href{https://www.dropbox.com/s/t9ic1rpirja4xqv/computational-exon-laderas.pdf?dl=0}{\emph{Computational
  detection of alternative exon usage}.} 8 citations, Impact Factor:
  3.398. \textbf{Ted Laderas}, Nicole A Walter, Michael Mooney, Kristina
  Vartanian, Priscila Darakjian, Kari Buck, Chris Harrington, John
  Belknap, Robert Hitzemann, and Shannon McWeeney. Frontiers in
  Neurogenomics. 2011. Article 69. PMID 21625610. Contribution: First
  author, developer of methodology.
  \href{http://dx.doi.org/10.3389/fnins.2011.00069}{doi:10.3389/fnins.2011.00069}
\item
  \href{https://www.dropbox.com/s/2v0h7ufxu1664yt/preponderance-SNPs-walter-laderas.pdf?dl=0}{\emph{High
  throughput sequencing in mice: a platform comparison identifies a
  preponderance of cryptic SNPs}.} 19 Citations, Impact Factor 3.87.
  Nicole A Walter, Daniel Bottomly, \textbf{Ted Laderas}, Michael
  Mooney, Priscila Darakjian, Robert P Searles, Christina Harrington,
  Shannon K McWeeney, Robert Hitzemann, Kari J Buck. BMC Genomics. 2009
  Aug 17;10:379. Contribution: helped develop methodology of manuscript,
  conceptualization.
  \href{http://dx.doi.org/10.1186/1471-2164-10-379}{doi:10.1186/1471-2164-10-379}
\item
  \emph{TandTRAQ: An open-source tool for integrated protein
  identification and quantitation}. 14 citations, Impact Factor 5.766.
  \textbf{Ted Laderas}, Cory Bystrom, Debra McMillen, Guang Fan and
  Shannon McWeeney. Bioinformatics. 2007. Contribution: first author,
  designed software framework.
  \href{http://dx.doi.org/10.1093/bioinformatics/btm467}{doi:10.1093/bioinformatics/btm467}
\item
  \href{https://www.dropbox.com/s/v56bomf1ijhmds3/Laderas_McWeeney07.pdf?dl=0}{\emph{A
  consensus framework for exploring microarray data using multiple
  clustering methods}.} 16 citations. \textbf{Ted Laderas} and Shannon
  McWeeney. OMICS: A Journal of Integrative Biology. OMICS: A Journal of
  Integrative Biology. 2007. 116-128. Contribution: first author,
  developed methodology for comparison analysis.
  \href{http://dx.doi.org/10.1089/omi.2006.0008}{doi:10.1089/omi.2006.0008}
\end{enumerate}

\subsection{Software Development}\label{software-development}

Sole Developer unless specified.

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \href{https://laderast.github.io/flowDashboard/}{\texttt{flowDashboard}}.
  Visualization framework in R/Shiny and processing pipeline for CyTOF
  and flow cytometry data. \url{https://doi.org/10.5281/zenodo.260421}
\item
  \href{https://laderast.github.io/cvdRiskData/}{\texttt{cvdRiskData}}.
  Synthetic dataset for teaching fundamentals in data science. Available
  as an R package at: \url{https://github.com/laderast/cvdRiskData}.
\item
  \href{https://github.com/laderast/dataLiteracyTutorial}{\texttt{dataLiteracyTutorial}}.
  \texttt{LearnR} package for teaching the fundamentals of graph
  literacy. Based on \texttt{DSIExplore} (written with Jessica Minnier)
  and
  \href{https://github.com/laderast/wmf-allhands18}{\texttt{wmf-allhands18}}
  (by Mikhail Popov).
\item
  \href{https://github.com/laderast/DSIExplore}{\texttt{DSIExplore}}.
  \texttt{LearnR} package for teaching Exploratory Data Analysis and
  Simple Statistics. Jessica Minnier and \textbf{Ted Laderas}.
\item
  \href{https://www.datacamp.com/courses/rbootcamp}{\emph{RBootcamp
  (tidyverse version)}}. DataCamp active exercises for learning
  introductory R using the \texttt{tidyverse}. Jessica Minnier and
  \textbf{Ted Laderas}. Available for free at:
  \url{https://www.datacamp.com/courses/rbootcamp}
  \url{https:://doi.org/10.5281/zenodo.854727}
\item
  \href{https://github.com/ropenscilabs/umapr}{\texttt{umapr}}. R
  Wrapper package for python implementation. Malisa Smith, Angela Li, Ju
  Young Kim, \textbf{Ted Laderas}, and Sean Hughes. Contribution:
  documentation and visualization framework for results.
\item
  \href{http://infer.netlify.com}{\texttt{infer}}. Andrew Bray, Chester
  Ismay, Ben Baumer, Mine Cetinkaya-Rundel, \textbf{Ted Laderas}, Nick
  Solomon, Johanna Hardin, Albert Kim, Neal Fultz and Doug Friedman.
  Contribution: package testing.
\item
  \emph{Surrogate Network Explorer for Head and Neck Cancer}.
  Interactive visualization framework for understanding network effects
  in Head and Neck cancer. Awaiting validation, publication, and
  submission.
\item
  \emph{R-Bootcamp: Introduction to R}. Script-based introductory MOOC
  to R. With contributions from Eric Leung, Dian Chase, Tracy Edinger,
  Clint Olson, and Gabrielle Chonoo.
  \url{http://dx.doi.org/10.5281/zenodo.13756}
\item
  \emph{Surrogate Mutation Explorer}. R/Shiny Interactive Application
  for exploring surrogate mutations.
  \url{http://dx.doi.org/10.5281/zenodo.13757}
\item
  \emph{SurrogateMutation}. R Package for mapping mutations and copy
  number alterations to networks and associated statistics.
  \url{http://dx.doi.org/10.5281/zenodo.17303}
\item
  \emph{Consense-Cluster}. R Package for comparing clustering output
  across algorithms. \url{http://dx.doi.org/10.5281/zenodo.17304}
\item
  \emph{ExonModelStrain}. RPackage for detecting alternative exon usage
  between two strains of mice.
  \url{http://dx.doi.org/10.5281/zenodo.17305}
\item
  \emph{TandTRAQ}. Perl Script for mapping iTRAQ protein quantitation to
  X!Tandem peptides.
\end{enumerate}

\subsection{Research Presentations/Lectures (Invited Lectures are
Indicated)}\label{research-presentationslectures-invited-lectures-are-indicated}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \href{https://ww2.amstat.org/meetings/jsm/2018/onlineprogram/AbstractDetails.cfm?abstractid=328720}{\emph{Mixing
  Active Learning and Lecturing: Using Interactive Visualization as a
  Teaching Tool}}. Presentation at the Joint Statistical Meeting. July
  2018. Jessica Minnier and \textbf{Ted Laderas}.
\item
  \href{http://laderast.github.io/talks/synth-data-laderas.pdf}{cvdRiskData:
  A synthetic patient cohort for teaching predictive modeling and data
  science}. Presentation/Interactive demonstration at AMIA Informatics
  Educators Forum. June 2018.
\item
  \href{https://www.causeweb.org/cause/ecots/ecots18/posters/3-03}{\emph{Mixing
  Active Learning and Lecturing: Using Interactive Visualization as a
  Teaching Tool}}. Virtual Poster given for ECOTS 2018 (Electronic
  Conference On Teaching Statistics). May 2018. Jessica Minnier and
  \textbf{Ted Laderas}.
\item
  \href{http://laderast.github.io/DS4BS/}{\emph{Data Science for Basic
  Scientists}}. Invited talk for OHSU Symposium on Educational
  Excellence. April 2017.
  \href{https://doi.org/10.6084/m9.figshare.4876811.v3}{doi:10.6084/m9.figshare.4876811.v3}
\item
  \href{https://www.dropbox.com/s/ls6agg9uy2b6wp4/big-data-talk_laderas.pdf?dl=0}{\emph{Moving
  From Big Data to Knowledge: Implications for the Health Care and
  Biomedical Sciences}}\emph{.} Invited Lecture given for BioCatalyst
  Advanced Training workshop at Oregon Bioscience Association. June
  2015.
\item
  \emph{Surrogate Mutations and Drug Sensitivity in Breast Cancer Cell
  Lines.} Plenary talk at NLM Training Conference, Pittburgh, PA, June
  2014.
\item
  \emph{Connecting Genotypes to Drug Sensitivities in HER2 Positive
  Cancer Cell Lines.} Doctoral Defense, OHSU, March 2014.
\item
  \emph{Understanding Expression Differences in RPPA Data: Implications
  for ODE Models.} Presentation at Integrative Cancer Biology Program
  retreat 2012.
\item
  \emph{Connecting Genotypes to Drug Sensitivity in HER2+ Breast Cancer
  Cell Lines.} Lecture/Dissertation proposal given for PhD/Postdoc
  Meeting January 2012 and March 2012.
\item
  \emph{Connecting Genotypes to Drug Sensitivity in Breast Cancer Cell
  Lines.} Invited presentation at Integrative Cancer Biology Program
  retreat, 2011.
\item
  \href{https://www.dropbox.com/s/bi14mo1ahr4yod8/symposium-laderas.pptx?dl=0}{\emph{What
  are Models For? Making Sense of Systems Biology}}. Predoctoral
  Symposium. 2010.
\item
  \emph{Two kinds of Robustness.} Presentation given to Bioinformatics
  Development Group, 2010.
\item
  \emph{Portland Alcohol Research Center Scientific Advisory Board
  Review: Bioinformatics \& Statistics.} Presentation given to PARC SAB,
  OHSU, 2009.
\item
  \emph{An introduction to trend analysis and consensus modules for
  systems biology.} Presentation given with Sophia Jeng to Mathematical
  Modeling Core for Systems Virology, OHSU 2009.
\item
  \emph{An introduction to Consensus Modules for Systems Biology.}
  Lecture given with Sophia Jeng to Bioinformatics Development Group,
  OHSU, 2008.
\item
  \emph{An Introduction to Sloppy Systems Biology.} Lecture given to
  Bioinformatics Development Group, OHSU, 2008.
\item
  \emph{Simulated and Synthetic Data for Benchmarking Algorithms.}
  Lecture given as part of Bioinformatics Methods class, OHSU, 2007.
\item
  \emph{Introduction to QPACA Pathway Analysis Software and BioPAX.}
  Talk given at 2006 caBIG Annual Meeting, Washington DC.
\item
  \emph{Consense: A software package for utilizing multiple clustering
  methods on Microarray Data.} Talk given at OHSU Student Research
  Forum, 2005.
\end{enumerate}

\subsection{Research Posters/Abstracts}\label{research-postersabstracts}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \href{http://laderast.github.io/talks/flowDashboard.pdf}{The
  \texttt{flowDashboard} package for comparative flow cytometry
  analysis.} Ted Laderas, Gwendolyn Swarbrick, David Lewinsohn, Debbie
  Lewinsohn, Evan Lind, and Shannon McWeeney. BOSC2018. June 2018.
\item
  \emph{Interactive Visualization as a Tool to Teach Mathematical
  Concepts to OHSU Data Science Institute Students}. Jessica Minnier and
  \emph{Ted Laderas}. OHSU Symposium on Educational Excellence. April
  2018.
\item
  \emph{Getting your hands dirty with data}. \textbf{Ted Laderas},
  Melissa Haendel, Bjorn Pederson, Jackie Wirz, William Hersh, David A.
  Dorr, Shannon McWeeney. Big Data to Knowledge (BD2K) All Hands
  meeting, November 2016.
  \href{https://doi.org/10.6084/m9.figshare.4235594.v1}{doi:10.6084/m9.figshare.4235594.v1}
\item
  \emph{Get Real: A synthetic dataset illustrating clinical and genetic
  covariates.} \textbf{Ted Laderas}, Nicole Vasilevsky, Melissa Haendel,
  Shannon McWeeney and David A. Dorr. BD2K All Hands meeting, November
  2016.
  \href{https://doi.org/10.6084/m9.figshare.4239959.v1}{doi:10.6084/m9.figshare.4239959.v1}
\item
  \emph{Mass Cytometry as a Modality to Identify Candidates for Immune
  Checkpoint Inhibitor Therapy within Acute Myeloid Leukemia.} Adam
  Lamble, Fei Huang, Kate Sasser, Homer Adams III, Cristina E. Tognon,
  \textbf{Ted Laderas}, Shannon McWeeney, Marc Loriaux, Jeffery W.
  Tyner, Brian J. Druker, and Evan F. Lind. American Society for
  Hematology, December 2016.
\item
  \emph{Enhanced VISTA Expression in a Subset of Patients with Acute
  Myeloid Leukemia}. Adam Lamble, Yoko Kosaka, Fei Huang, Kate Sasser,
  Homer Adams III, Cristina E. Tognon, \textbf{Ted Laderas}, Shannon
  McWeeney, Marc Loriaux, Jeffery W. Tyner, Brian J. Druker, and Evan F.
  Lind. American Society for Hematology, December 2016.
\item
  \emph{Improving Knowledge Discovery Through Development of Big Data to
  Knowledge Skills Courses and Open Educational Resources}. Nicole
  Vasilevsky, Shannon McWeeney, William Hersh, David A. Dorr,
  \textbf{Ted Laderas}, Jackie Wirz, Bjorn Pederson and Melissa Haendel.
  Poster for BICC Building 25th Anniversary.
\item
  \emph{Consensus molecular subtyping through a community of experts
  advances unsupervised gene expression-based disease classification and
  facilitates clinical translation.} Justin Guinney, Rodrigo Dienstmann,
  Xin Wang, Aurelien de Reynies, Andreas Schlicker, Charlotte Soneson,
  Laetitia Marisa, Paul Roepman, Gift Nyamundanda, Paolo Angelino, Brian
  Bot, Jeffrey S. Morris, Iris Simon, Sarah Gerster, Evelyn Fessler,
  Felipe de Sousa e Melo, Edoardo Missiaglia, Hena Ramay, David Barras,
  Krisztian Homicsko, Dipen Maru, Ganiraju Manyam, Bradley Broom,
  Valerie Boige, \textbf{Ted Laderas}, Ramon Salazar, Joe W. Gray, Josep
  Tabernero, Rene Bernards, Stephen Friend, Pierre Laurent-Puig, Jan P.
  Medema, Anguraj Sadanandam, Lodewyk Wessels, Mauro Delorenzi, Scott
  Kopetz, Louis Vermeulen and Sabine Tejpar. American Association for
  Cancer Research Conference, August 2015.
\item
  \emph{A Network-Based Model of Oncogenic Collaboration for Prediction
  of Drug Sensitivity.} \textbf{Ted Laderas}. RECOMB/ISCB Conference on
  Regulatory and Systems Genomics, November 2014.
\item
  \href{https://www.dropbox.com/s/y3rbkygjn41qsd6/poster-laderas-nlm-2013.pdf?dl=0}{\emph{Integrating
  proteomics and genomic data to illuminate the effects of Lapatinib in
  HER2+ Cells}}. \textbf{Ted Laderas} and Kemal Sonmez. National Library
  of Medicine Training Conference, Utah, 2013.
  \href{https://doi.org/10.6084/m9.figshare.4829749.v1}{doi:10.6084/m9.figshare.4829749.v1}
\item
  \href{https://www.dropbox.com/s/nhixsztlg1oo4z6/poster-laderas-ismb-2012.pdf?dl=0}{\emph{Integrating
  to Interact: Using Reconstructability Analysis to Integrate Genomic
  and Proteomic Data to Predict Drug Sensitivity}}. \textbf{Ted Laderas}
  and Kemal Sonmez. Intelligent Systems in Molecular Biology conference,
  2012.
  \href{https://doi.org/10.6084/m9.figshare.4830007.v1}{doi:10.6084/m9.figshare.4830007.v1}
\item
  \emph{Tales from the Cryptic Snp: High Throughput Sequencing of DBA/2J
  and C57BL/6J mouse chromosome 1 reveals an abundance of
  polymorphisms.} Kari J. Buck, Nicole A.R. Walter, Daniel Bottomly,
  \textbf{Ted Laderas}, Michael Mooney, Priscila Darakjian, Shannon
  McWeeney, Robert Hitzemann. Research Society on Alcoholism Annual
  Meeting. 2009.
\item
  \emph{A computational framework for the detection of alternative exon
  usage using Affymetrix Exon arrays.} \textbf{Ted Laderas}, Michael
  Mooney, Nikki Walter, Kristina Vartanian, Christina Harrington,
  Priscila Darakijan, John Belknap, Robert Hitzemann, and Shannon
  McWeeney. Intelligent Systems in Molecular Biology conference,
  Toronto, July 2008.
\item
  \emph{Cerebrospinal fluid (CSF) proteomics in children with acute
  lymphoblastic leukemia (ALL).} Robert C Trueworthy, Linda C Stork,
  Yanping Zhong, Sharon Pine, Yousif Matloub, \textbf{Ted Laderas},
  Shannon McWeeney, and Guang Fan. Poster 2006.
\item
  \emph{A metrics-based low-level preprocessing framework for proteomics
  data.} \textbf{Ted Laderas}, Solange Mongoue-Tchokote, Veena
  Rajaraman, Kimberly Sellers, Cory Bystrom and Shannon McWeeney.
  Bioconductor Developer's Meeting, Seattle, Washington, August 2006.
\item
  \emph{A diagnostic test suite for the RProteomics statistical
  routines.} \textbf{Ted Laderas}, Solange Mongoue-Tchokote, Veena
  Rajaraman, Kimberly Sellers and Shannon McWeeney. caBIG annual
  meeting, Washington DC, April 2006.
\item
  \emph{Consense-Cluster: a framework for comparing and benchmarking
  clustering algorithms.} \textbf{Ted Laderas} and Shannon McWeeney.
  Bioconductor Developer's Meeting, Seattle, Washington, August 2005.
\end{enumerate}

\section{SERVICE}\label{service}

\subsection{Service Statement}\label{service-statement}

I am a strong supporter of service at OHSU and beyond. I currently
participate in the DMICE BCB (Bioinformatics and Computational Biology)
Faculty Division meeting, the DMICE Mentoring committee, and have
participated in the BCB Curriculum Retreat in order to plan upcoming
coursework at DMICE. Being gay and a Pacific Islander, I have a unique
viewpoint about diversity. I have attempted to contribute to diversity
and inclusion by making workshops and organizing conferences that are
welcoming to diverse audiences. Beyond OHSU, I believe that we need to
increase public engagement of science and increase outreachand mentoring
of next-generation science students, especially from disadvantaged
populations will enable these students to succeed in STEM-based careers.
As a former student of Saturday Academy's scientific mentoring program,
I want to contribute back to this community and engage potential STEM
students through student outreach and mentoring. I am also involved in
outreach through the development of course material for the Biocatalyst
training program through Oregon Bioscience Association, which provides
bioscience training for unemployed or under-employed professionals.

\subsection{Service
Accomplishments/Appointments}\label{service-accomplishmentsappointments}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  Currently serving on

  \begin{itemize}
  \tightlist
  \item
    DMICE Assessment Planning committee
  \item
    DMICE BCB Division committee
  \item
    DMICE BMI Curriculum Committee
  \item
    DMICE Mentorship Committee
  \end{itemize}
\item
  Mentor for DMICE Fellows Meeting Fall 2016, and Winter 2018.
\item
  Co-Founder and Organizer of
  \href{https://biodata-club.github.io}{BioData Club} group for
  students, postdocs, and staff at OHSU. Managed speakers, developed
  free workshops for students, and provided feedback for student
  presentations. BioData Club is currently at 109 members ranging from
  OHSU and PSU staff, faculty, students and postdocs, providing a unique
  opportunity to share information and provide peer mentoring to
  students across the organizations.
\item
  Co-organizer for \href{https://cascadiarconf.com}{Cascadia R
  Conference} 2017 and 2018. Logistics, planning, assisted with
  selection and diversity committee.
\item
  Contributor to instructor lesson materials to
  \href{https://software-carpentry.org}{Software Carpentry}.
\item
  Editor at Large for \href{http://radiandata.org/}{Radian Data}.
\item
  Participant/Instructor for
  \href{https://github.com/daniellecrobinson/Data-Rescue-PDX}{PDX Data
  Rescue/Open Data Day 2017}.
\item
  Documentation support for
  \href{http://portland.sciencehackday.org}{Science Hack Day PDX 2017}.
\item
  Developed coursework for unemployed or underemployed IT workers in
  both biosciences and healthcare for Oregon Bioscience Association.
\item
  Reviewer for:

  \begin{itemize}
  \tightlist
  \item
    Journal of the American Medical Informatics Association (Open
    Reviewer)
  \item
    Gates Open Research (Open Reviewer)
  \item
    BMC Bioinformatics
  \item
    Neurocomputing (Co-Reviewer)
  \item
    Bioinformatics (Open Reviewer)
  \item
    AMIA Symposium (Open Reviewer)
  \end{itemize}
\item
  My mentees include:

  \begin{itemize}
  \tightlist
  \item
    Jason Li (Mentor)
  \item
    Eisa Mahyari (Co-mentor)
  \item
    Lawrence Hsu (Co-mentor)
  \item
    Gabrielle Choonoo (Thesis committee member)
  \item
    Connor Smith (Thesis Committee)
  \item
    Sean Babcock (Thesis Committee)
  \end{itemize}
\end{enumerate}

\section{Work Experience}\label{work-experience}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \emph{Assistant Professor}, Department of Medical Informatics and
  Clinical Epidemiolgy, OHSU, 2017-present.
\item
  \emph{Postdoctoral Researcher}, OHSU Knight Cancer Institute,
  2014-2017. \emph{Instructor}, Department of Medical Informatics and
  Clinical Epidemiology. 2015-Present.
\item
  \emph{NLM Postdoctoral Fellow}, Oregon Health \& Science University.
  2014-2015.\\
  \emph{Visiting Scientist}, Sage Bionetworks, Seattle Washington.
  2014-2015.
\item
  \emph{NLM Predoctoral Fellow}, Medical Informatics and Clinical
  Epidemiology, Oregon Health \& Science University. 2009--2014.
\item
  \emph{Bioinformatics Developer/Project Manager}, OHSU Knight Cancer
  Institute, Oregon Health \& Science University. 2003-2009.
\item
  \emph{Teaching Assistant/Computer Programmer/Server Admin}, Department
  of Medical Informatics \& Clinical Epidemiology, Oregon Health \&
  Science University. 2001-2002.
\item
  \emph{Research Assistant/Computer Programmer}, Department of Molecular
  Medicine, Oregon Health \& Science University. 1999-2001.
\item
  \emph{Research Assistant/Teaching Assistant}, Department of Chemistry,
  Reed College. 1998
\end{enumerate}

\section{Ongoing Research Support}\label{ongoing-research-support}

T15LM009461 (supplement)\\
Hersh (PI)\\
7/01/1992-6/01/2018

\emph{Biomedical Informatics Research Training at Oregon Health \&
Science University.} Supplement to develop Data Science Materials for
T15 training grant. Role: educational developer and instructor.

1U24TR002306-01\\
Haendel (PI)\\
9/01/2017-9/01/2019

\emph{A National Center for Digital Health Informatics Innovation}. We
propose to create a national network for enabling digital health
research, innovation, and continuous improvement. The goal is to use
information science to impact the way that health care functions and the
lives of those it serves. Role: education, software development, and
data management advocate.

1U54CA224019-01\\
Tyner (PI)\\
10/01/2017-10/01/2022

\emph{Tumor intrinsic and microenvironmental mechanisms driving drug
combination efficacy and resistance in AML}. Most patients with acute
myeloid leukemia (AML) eventually die when their disease becomes
resistant to conventional or even newer treatments. Our proposed studies
will shed light on the mechanisms of drug resistance, both within the
tumor and in the surrounding environment. This knowledge will help
identify more effective therapies --- involving combinations of two
drugs --- that will avoid drug resistance and provide better outcomes
for patients with AML. Role: Computational Biologist.

OPP1131709\\
Lewinsohn (PI)\\
10/26/2015-10/31/2018 Bill \& Melinda Gates Foundation

\emph{Targeting MAIT cells for TB vaccines}. This proposal is designed
to establish whether or not a vaccine targeting Mucosal Associated
Invariant (MAIT) can be used to prevent tuberculosis (TB). Role:
Computational Biologist

\section{Completed Research Support}\label{completed-research-support}

No \# assigned Druker (PI) 5/1/2013-5/31/2018 The Leukemia \& Lymphoma
Society \emph{Beat AML: Precision Medicine for AML Based on Functional
Genomics}. The major goals of this project is to transform our approach
to AML treatment through a deeper understanding of the diversity of the
underlying molecular causes of disease and to bring targeted therapies
to AML patients through 1) understanding the spectrum of genetic lesions
and molecular drivers, 2) functionally annotating drug sensitivity, and
3) Initiating clinical trials with combinations of drugs in refractory
patients. Role: Computational Biologist.

T15LM009461\\
Hersh (PI)\\
7/01/1992-6/30/2017\\
\emph{Biomedical Informatics Research Training at Oregon Health \&
Science University}. Predoctoral and postdoctoral training grant in
biomedical informatics. Role: Predoctoral Fellow (2009-14); Postdoctoral
Fellow (2014-2015)

BD2K Training Grant Dorr, D (Co-PI), Haendel, M (Co-PI), McWeeney, S
(Co-PI) 2015-2017 \emph{Big Data to Knowledge}. The goal of this project
is to develop training materials in data science. Role: educational
developer and instructor.

\subsection{AWARDS}\label{awards}

NLM Predoctoral Fellowship. 2009-2013.\\
NLM Postdoctoral Fellowship. 2014-2015.


\end{document}
